src/io/sampling.jl

```diff
@@ -65,24 +65,68 @@ end
 
 
 """
-    sample(dom::Domain, n::Int, sampler=SobolSample())::DataFrame
+    sample(dom::Domain, n::Int)::DataFrame
```

Comment:
Ability to specify a sampler removed?

Reply:
Not intentional, this may have been left over from a structural change I tried which I then reverted. I'm make sure this hasn't happened anywhere else